src/triage/component/architect/builders.py

```diff
@@ -268,15 +281,22 @@ def build_matrix(
         logging.info('Creating matrix %s > %s', matrix_metadata['matrix_id'], matrix_filename)
         # make the entity time table and query the labels and features tables
         logging.info('Making entity date table for matrix %s', matrix_uuid)
-        entity_date_table_name = self.make_entity_date_table(
-            as_of_times,
-            label_name,
-            label_type,
-            matrix_metadata['state'],
-            matrix_type,
-            matrix_uuid,
-            matrix_metadata['label_timespan']
-        )
+        try:
+            entity_date_table_name = self.make_entity_date_table(
+                as_of_times,
+                label_name,
+                label_type,
+                matrix_metadata['state'],
+                matrix_type,
+                matrix_uuid,
+                matrix_metadata['label_timespan']
+            )
+        except ValueError as e:
+            logging.warning(
+                'Not able to build entity-date table due to: %s - will not build matrix',
+                str(e)
```

Comment:
rather than just the error message, do you want to provide the traceback here? in that case you don't need to assign the exception object, just:

    logging.warning(.., exc_info=True)